app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<%=stylesheet_link_tag('devise')%>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'sign-in-form' }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'form_field'  %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autocomplete: "email",placeholder: "User Name", class: 'form_field'  %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password",placeholder: "Password" class: 'form_field'  %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'form_field'  %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, size: "autox10", autocomplete: "bio", class: 'form_field'  %>
+  </div>
+
+  <%= f.hidden_field :posts_counter, value: 0 %>
+  <%= f.hidden_field :photo, value: 'https://unsplash.com/photos/F_-0BxGuVvo' %>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class: 'form_field'  %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links"%>
```

Comment:
- [ ] Great job adding devise to your app. However, I am unable to register a new user as I got this error message while trying to signup. :point_down: 

### Error Message
![error](https://github.com/haftamudesta/Blog-App/assets/26597432/9ae7fa1d-63bd-4bb1-810b-1db3391e6668)

### Screen Record
[app.webm](https://github.com/haftamudesta/Blog-App/assets/26597432/3d9d51d2-e880-4938-9987-8c87c9d2a2d1)

Kindly look into it and fix.